core/templates/dev/head/player/learner_local_nav.html

```diff
@@ -1,5 +1,12 @@
 <ul class="nav navbar-nav oppia-navbar-nav navbar-right"
     ng-controller="LearnerLocalNav" style="margin-right: 0px;">
+  <li>
+    {% if username %}
+      <a ng-click="showFlagExplorationModal()" tooltip="Flag Exploration" tooltip-placement="bottom">
```

Comment:
I tried to internationalize it using google translate. Let me know if it is good.